jdaviz/configs/default/plugins/data_quality/data_quality.py

```diff
@@ -0,0 +1,285 @@
+
+    flag_map_definitions = Dict().tag(sync=True)
+    flag_map_selected = Any().tag(sync=True)
+    flag_map_definitions_selected = Dict().tag(sync=True)
+    flag_map_items = List().tag(sync=True)
+    viewer_selected = Any().tag(sync=True)  # Any needed for multiselect
+    decoded_flags = List().tag(sync=True)
+    flags_filter = List().tag(sync=True)
+
+    icons = Dict().tag(sync=True)
+    icon_radialtocheck = Unicode(read_icon(os.path.join(ICON_DIR, 'radialtocheck.svg'), 'svg+xml')).tag(sync=True)  # noqa
+    icon_checktoradial = Unicode(read_icon(os.path.join(ICON_DIR, 'checktoradial.svg'), 'svg+xml')).tag(sync=True)  # noqa
+
+    def __init__(self, *args, **kwargs):
+        super().__init__(*args, **kwargs)
+
+        self.icons = {k: v for k, v in self.app.state.icons.items()}
+
+        self.viewer = ViewerSelect(
+            self, 'viewer_items', 'viewer_selected', 'viewer_multiselect'
+        )
+        self.science_layer = LayerSelect(
+            self, 'science_layer_items', 'science_layer_selected',
+            'viewer_selected', 'science_layer_multiselect', is_root=True
+        )
+
+        self.dq_layer = LayerSelect(
+            self, 'dq_layer_items', 'dq_layer_selected',
+            'viewer_selected', 'dq_layer_multiselect', is_root=False,
+            is_child_of=self.science_layer.selected
+        )
+
+        self.load_default_flag_maps()
+        self.init_decoding()
+
+    @observe('science_layer_selected')
+    def update_dq_layer(self, *args):
+        if not hasattr(self, 'dq_layer'):
+            return
+
+        self.dq_layer.filter_is_child_of = self.science_layer_selected
+        self.dq_layer._update_layer_items()
+
+        # listen for changes on the image opacity, and update the
+        # data quality layer opacity on changes to the science layer opacity
+        plot_options = self.app.get_tray_item_from_name('g-plot-options')
+        plot_options.observe(self.update_opacity, 'image_opacity_value')
+
+    def load_default_flag_maps(self):
+        for name in dq_flag_map_paths:
+            self.flag_map_definitions[name] = load_flag_map(name)
+            self.flag_map_items = self.flag_map_items + [telescope_names[name]]
+
+    @property
+    def multiselect(self):
+        logging.warning(f"DeprecationWarning: multiselect has been replaced by separate viewer_multiselect and layer_multiselect and will be removed in the future.  This currently evaluates viewer_multiselect or layer_multiselect")  # noqa
+        return self.viewer_multiselect or self.layer_multiselect
+
+    @multiselect.setter
+    def multiselect(self, value):
+        logging.warning(f"DeprecationWarning: multiselect has been replaced by separate viewer_multiselect and layer_multiselect and will be removed in the future.  This currently sets viewer_multiselect and layer_multiselect")  # noqa
+        self.viewer_multiselect = value
+        self.layer_multiselect = value
```

Comment:
do we need these since this is a new plugin (I'm assuming this was just copied directly from plot options)?  Or do you want a single multiselect instead of separate layer and viewer options?